reflex/ivars/sequence.py

```diff
@@ -933,6 +1314,62 @@ def array_reverse_operation(
     )
 
 
+@var_operation
+def array_lt_operation(lhs: ArrayVar | list | tuple, rhs: ArrayVar | list | tuple):
```

Comment:
these `array_*_operation` functions seem really similar to the string flavored functions, are they different just due to the annotations? or is there some other difference i'm missing

Reply:
it's only annotations wise, we could in theory merge the two although the functions would kinda except anything (which could be ok)